server/app.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+const http = require('http');
+const express = require('express');
+const bodyParser = require('body-parser');
+const _ = require('lodash');
+const fs = require('fs');
+const usersRouter = require('./routes/user');
+
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost:27017/BePreparedApp');
+const {User} = require('./DataBase/models/user');
+const {Course} = require('./DataBase/models/course');
+
+const publicPath = path.join(__dirname , '../public');
```

Comment:
do we have anything in public right now??

Reply:
No not right now. But this is for the connection between frontend and the backend